### ISSUE TITLE
Mod startup animation and sound [new feature request]

### ISSUE THREAD

**Issue by MoritzWoerle** [opera-gaming/gxmods#1]
Would it be possible to mod the startup animation and sound?
I would really like to do this, to match it to my theme.

**Comment by DrKonekoReal** [comment]
Good idea, this will likely happen becuase of the Opera GF animation that was avaliable today.

**Comment by zyber-sudo** [comment]
How would one adjust the size in the .json?
